syft/serde.py

```diff
@@ -46,12 +46,30 @@
 
 from .frameworks.torch.tensors.abstract import initialize_tensor
 
+
+# COMPRESSION SCHEMES
+LZ4 = 0
```

Comment:
Not a priority, but maybe make this a enum?

Reply:
We've debated this a little bit - apparently enums are a bit less efficient for (more or less) the same interface (@LaRiffle did a few tests on the 0.3.1 version).

Reply:
Interesting, thanks for explaining.